rocAL/rocAL/source/pipeline/image.cpp

```diff
@@ -132,7 +132,7 @@ ImageInfo::ImageInfo(
         _height(height_),
         _color_planes(planes),
         _batch_size(batches),
-        _data_size((static_cast<uint64_t>(width_ * height_ * _batch_size * planes))),
+        _data_size((static_cast<uint64_t>(width_ * height_ * _batch_size * planes / 8) * 8 + 8)),
```

Comment:
Why are we doing this?

Reply:
@sampath1117 Can u explain why this patch is needed here?